docs/appendix/infrastructure-template.md

```diff
@@ -61,18 +61,6 @@ export DISABLE_DOMAIN=true
 
 The Verify API creates an endpoint that accepts a TradeTrust document and returns the results of the verification according to the [@govtechsg/oa-verify](https://www.npmjs.com/package/@govtechsg/oa-verify) library.
 
-To deploy the Verify API to be used on Ropsten:
-
-```bash
-NETWORK=ropsten npm run deploy:verify -- --stage stg
-```
-
-To deploy the Verify API to be used on Rinkeby:
-
-```bash
-NETWORK=rinkeby npm run deploy:verify -- --stage stg
-```
-
```

Comment:
Same as below, is it because this feature still works on mainnet but not on testnets anymore?

Reply:
same will update accordingly